### PR TITLE
random(int m, int n) - #8

### LINKED ISSUE #8: random(int m, int n)
Generate matrix with random elements

### PR DESCRIPTION
**Pull request by KaraBeason**
Closes #8.  Function implemented. .... so I wrote a pretty trivial unit tests that just asserts random does not return a null Javatrix object.  I'm not sure how to unit test a function that sets a matrix's elements to something random ... as the constructor used and set function have already been unit tested by @haietza .  What are your thoughts?

### CODE REVIEW

**Comment by haietza** [comment]
We could iterate through the elements of the returned Javatrix object to make sure they are not all the same. However I'm not sure how valid that would be - Math.random could possibly return 2 values that were equal with all the others being different and it would still be random. I think the unit tests are more for testing expected values, and since randomness contradicts that, let's just proceed with what you have :).

File: Javatrix.java [hunk 1]
```diff
@@ -243,5 +243,26 @@ public static Javatrix constructWithCopy(double[][] arr)
...
+     * @return a Javatrix object with matrix of random elements
+     */
+    public static Javatrix random(int m, int n)
+    {
+        Javatrix randomMatrix = new Javatrix(m, n);
+        for (int i = 0; i < m; i++)
+        {
+            for (int j = 0; j < n; j++)
+            {
+                double rand = java.lang.Math.random();
```
Comment (haietza) [inline Javatrix.java]: Should we declare the rand variable outside the loop, then initialize its value inside, just so we're not creating a new variable for each iteration?

Comment (KaraBeason) [reply Javatrix.java]: Yes!  Fixing now, thank you!

**Comment by KaraBeason** [comment]
@haietza Okay, sounds good!  Should be good to go now.

**Review by haietza — APPROVED** [review]
Looks good!

**Pull request merged**